packages/shared/src/components/fields/CustomSwitch.tsx

```diff
@@ -68,60 +77,69 @@ export function CustomSwitch({
   const difference = Math.abs(rightWidth - leftWidth);
 
   return (
-    <label
-      className={classNames(
-        'group relative flex h-9 w-fit cursor-pointer font-bold typo-callout',
-        className,
-      )}
-      htmlFor={inputId}
-    >
-      <input
-        id={inputId}
-        name={name}
-        type="checkbox"
-        className="absolute h-0 w-0 opacity-0"
-        checked={checked}
-        onChange={onToggle}
-      />
-      <ContentContainer
-        ref={leftRef}
-        className={checked && 'text-text-tertiary'}
-      >
-        {typeof leftContent === 'string'
-          ? leftContent
-          : React.cloneElement(leftContent as ReactElement, {
-              className: classNames(
-                (leftContent as ReactElement).props.className,
-                leftClasses,
-              ),
-            })}
-      </ContentContainer>
-      <ContentContainer
-        ref={rightRef}
-        className={!checked && 'text-text-tertiary'}
-      >
-        {typeof rightContent === 'string'
-          ? rightContent
-          : React.cloneElement(rightContent as ReactElement, {
-              className: classNames(
-                (rightContent as ReactElement).props.className,
-                rightClasses,
-              ),
-            })}
-      </ContentContainer>
-      <span className="absolute inset-0 my-auto h-7 rounded-10 bg-raw-cabbage-50 opacity-24 group-hover:opacity-32" />
-      <span
+    <SimpleTooltip content={tooltip?.content} placement={tooltip?.placement}>
+      <label
         className={classNames(
-          'absolute  top-0 z-1 h-full rounded-12',
-          styles.knob,
+          'group relative flex h-9 w-fit cursor-pointer font-bold typo-callout',
+          className,
```

Comment:
I see we accept `className` here and below `props.className` can you check which one is wrong?